### ISSUE TITLE
Allow for namespaced formatters

### ISSUE THREAD

**Issue by cupakromer** [cupakromer/emoji-rspec#6]
Passing in a namedspaced class fails when setting the class constant: `'const_set': wrong constant name Bar::BazFormatter (NameError)`

``` text
emoji-rspec/lib/emoji_test_love/rspec/rspec_integration.rb:48:in `const_set': wrong constant name Bar::BazFormatter (NameError)
        from emoji-rspec/lib/emoji_test_love/rspec/rspec_integration.rb:48:in `RSpecFormatter'
        from emoji-rspec/spec/generating_formatters_spec.rb:37:in `<top (required)>'
        from gems/rspec-core-2.13.1/lib/rspec/core/configuration.rb:819:in `load'
        from gems/rspec-core-2.13.1/lib/rspec/core/configuration.rb:819:in `block in load_spec_files'
        from gems/rspec-core-2.13.1/lib/rspec/core/configuration.rb:819:in `each'
        from gems/rspec-core-2.13.1/lib/rspec/core/configuration.rb:819:in `load_spec_files'
        from gems/rspec-core-2.13.1/lib/rspec/core/command_line.rb:22:in `run'
        from gems/rspec-core-2.13.1/lib/rspec/core/runner.rb:80:in `run'
        from gems/rspec-core-2.13.1/lib/rspec/core/runner.rb:17:in `block in autorun'
```

**Comment by cupakromer** [comment]
I was thinking about this and see two approaches off hand:
1. Inspect the string creating sub modules/constants
2. Replace `'::'` with something else in the string.

Thoughts /cc @coreyhaines

**Comment by cupakromer** [comment]
Fixed per commit 39425f2339561b2025f55f7f81a732b83ed9d92d.
